packages/react-components/react-nav-preview/src/components/NavItem/useNavItemStyles.styles.ts

```diff
@@ -23,19 +37,44 @@ const useContentStyles = makeStyles({
   selected: typographyStyles.body1Strong,
 });
 
+const indicatorHeight = '20px';
```

Comment:
you could add this to the comment below about the vars in an object

Reply:
added :-)